mean/modules/survey/views/p17_30.html

```diff
@@ -0,0 +1,391 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
```

Comment:
I used a template to generate pages. Basically help complete the grunt work behind 30 pages. Should I delete ?